orc8r/tools/fab/hosts.py

```diff
@@ -32,6 +32,8 @@ def vagrant_setup(host, destroy_vm, force_provision=False):
 
     host: the Vagrant box to setup, e.g. "magma"
     """
+    # An empty local ssh config file stops warnings from being printed.
+    local('touch ~/.ssh/config')
```

Comment:
It might make more sense to put this into `setup_env_vagrant` where we call `vagrant ssh-config`.

Reply:
Thanks, yes this makes sense. I was not sure where to put it.